### PR TITLE
[14.0] dotfiles update needs manual intervention

### PR DESCRIPTION
**Pull request by etobella**
Dear maintainer,

After updating the dotfiles, `pre-commit run -a`
fails in a manner that cannot be resolved automatically.

Can you please have a look, fix and merge?

Thanks,

### CODE REVIEW

**Comment by francesco-ooops** [comment]
@pedrobaeza @etobella could you clarify if we are waiting for a decision on what is being discussed in the contributor mailing list or if this PR is ready to merge?

**Comment by pedrobaeza** [comment]
Someone should take this pull request and fix the problem (in this case, diff merge problem). The green CI is a fake one.

File: .github/workflows/pre-commit.yml [hunk 1]
```diff
@@ -13,10 +13,17 @@ jobs:
   pre-commit:
     runs-on: ubuntu-22.04
     steps:
+<<<<<<< before updating
```
Comment (etobella) [inline .github/workflows/pre-commit.yml:16]: This needs to be fixed

**Comment by HekkiMelody** [comment]
Working on this

**Pull request closed without merging**